.github/workflows/integration_test_calamari.yml

```diff
@@ -76,7 +76,8 @@ jobs:
       - name: init
         run: |
           sudo apt update
-          sudo apt install -y pkg-config libssl-dev
+          sudo apt install -y pkg-config libssl-dev protobuf-compiler
```

Comment:
Not sure why to install `protobuf`.

Reply:
CI error in this old report: https://github.com/Manta-Network/Manta/actions/runs/4302144494/jobs/7500246339